lib/Resque.pm

```diff
@@ -97,7 +101,7 @@ has redis => (
     is      => 'ro',
     lazy    => 1,
     coerce  => 1,
-    isa     => 'Sugar::Redis',
+    isa     => 'Sugar::Redis|Object',
```

Comment:
I agree with you!

I must admit, I wasn't familiar with duck_type.  But I did think that having "something|Object"  does look a bit odd!

I'll try it duck_type, and upload an update assuming it works out OK.